File: Domain/AppNotificationsMaster.cs
using System;

namespace Domain
{
    public class AppNotificationsMaster
    {
		public int Id { get; set; }
		public string Subject { get; set; }
		public string Body { get; set; }
		public DateTime NextReminderTime { get; set; }
		public int NoOfReminders { get; set; }
		public int Frequency { get; set; }
		public string Type { get; set; }
		public int DataId { get; set; }
		public string ToUsers { get; set; }
		public string CCUsers { get; set; }
		public string ToGroups { get; set; }
		public string CCGroups { get; set; }
    }
}

File: client-app/src/features/AppNotificationsMaster/AppNotificationsMaster.ts
export interface IAppNotificationsMaster {
	Id: number
	Subject: string
	Body: string
	NextReminderTime: Date
	NoOfReminders: number
	Frequency: number
	Type: string
	DataId: number
	ToUsers: string
	CCUsers: string
	ToGroups: string
	CCGroups: string
}

export class AppNotificationsMaster implements IAppNotificationsMaster {
	Id: number = 0;
	Subject: string = '';
	Body: string = '';
	NextReminderTime: Date = new Date();
	NoOfReminders: number = 0;
	Frequency: number = 0;
	Type: string = '';
	DataId: number = 0;
	ToUsers: string = '';
	CCUsers: string = '';
	ToGroups: string = '';
	CCGroups: string = '';
  
  constructor(init?: IAppNotificationsMaster) {
    Object.assign(this, init);
  }
}

